python/py-pdbx/Portfile

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
+
+name                py-pdbx
+version             2.0.1
+revision            0
+
+checksums           rmd160  68a545db541a66af24eece3a60050be437199b00 \
+                    sha256  ad64a02a59d41e0013554ba03153d2ecbcb55579453b49be89c87ed228d0d1b1 \
+                    size    2149838
+
+supported_archs     noarch
+platforms           {darwin any}
+license             CCO-1.0
+maintainers         {gmail.com:howarth.at.macports @jwhowarth} openmaintainer
+
+description         Utilities for PDBx/mmCIF storage model
+long_description    {*}${description}
+
+github.setup            Electrostatics    mmcif-pdbx    ${version} v
```

Comment:
as I requested before, please us this as intended. No need to hardcode the version above, just do it here as all other ports do that use the `github` PortGroup. Also, just separate by a single space, no need for this formatting.

Reply:
also the project name is incorrect, it should be `mmcif_pdbx`